spec/rails_helper.rb

```diff
@@ -17,7 +17,7 @@
 Sidekiq::Testing.fake!
 
 # Stub Geocoder HTTP requests in specs
-Geocoder::DEFAULT_STUB_COORDINATES = [51.67014192630465, -1.2809649516211556].freeze
+Geocoder::DEFAULT_STUB_COORDINATES = [51.1789, -1.8262].freeze
```

Comment:
This is used in geocoding#coordinates `return self.class.test_coordinates if Rails.application.config.geocoder_lookup == :test`.

I used the coordinates for stonehenge as it is a major landmark and much easier to quickly validate distances from than some street in a place called Abingdon which the previous coordinates referred to.

Reply:
Awesome, thanks for the explanation.